apidoc/Makefile.am

```diff
@@ -119,7 +119,19 @@ include $(top_srcdir)/gtk-doc.make
 
 EXTRA_DIST += \
 	version.xml \
+	ostree-main-sections.txt \
+	ostree-experimental-sections.txt \
+	$(NULL)
+CLEANFILES += \
 	ostree-sections.txt \
 	$(NULL)
 
+if ENABLE_EXPERIMENTAL_API
+ostree-sections.txt: ostree-main-sections.txt ostree-experimental-sections.txt
+	cat $^ > $@
```

Comment:
This is quite minor but I habitually make build rules write files atomically, otherwise you can end up with weird, hard to debug problems if a build is interrupted in the middle.  So:

```
cat  $^ > $@.tmp && mv $@{.tmp,}
```

(Perhaps in some future world Unix cmdline tools will learn `O_TMPFILE`)